restapi/user_buckets.go

```
@@ -606,14 +641,15 @@ func getBucketInfo(ctx context.Context, client MinioClient, adminClient MinioAdm
 	if bucketAccess == models.BucketAccessPRIVATE && policyStr != "" {
 		bucketAccess = models.BucketAccessCUSTOM
 	}
-
+	bucketTags, _ := client.GetBucketTagging(ctx, bucketName)
```

you are ignoring the error, please handle it, if there's no tags, `bucketTags` is `nil` and it causes an NPE crashing the whole request